app/tandem/SEAS.cpp

```diff
@@ -135,16 +135,51 @@ template <typename T> struct qd_operator_specifics {
 
     static void print_profile(T const&) {}
 };
+
 template <>
-struct operator_specifics<SeasQDOperator> : public qd_operator_specifics<SeasQDOperator> {};
+struct operator_specifics<SeasQDOperator>
+  : public qd_operator_specifics<SeasQDOperator> {};
+
+
+template <typename T> struct qd_green_operator_specifics {
+  using monitor_t = seas::MonitorQD;
+
+  static auto make(LocalSimplexMesh<DomainDimension> const& mesh, Config const& cfg, seas::ContextBase& ctx) {
+    std::string prefix;
+    if (cfg.gf_checkpoint_prefix) { prefix = *(cfg.gf_checkpoint_prefix); }
+    else { prefix = ""; }
```

Comment:
This doesn't make a lot of sense as you could just pass on the optional. In QDGreen operator you check whether prefix is empty but you might as well check whether an std::optional prefix is given.

Reply:
now using std::optional prefix